staging/src/k8s.io/mount-utils/mount_linux.go

```diff
@@ -196,6 +196,7 @@ func (mounter *Mounter) doMount(mounterPath string, mountCmd string, source stri
 		return fmt.Errorf("mount failed: %v\nMounting command: %s\nMounting arguments: %s\nOutput: %s",
 			err, mountCmd, mountArgsLogStr, string(output))
 	}
+	klog.V(4).Infof("Volume successfully mounted, output: (%s)", string(output))
 	return err
```

Comment:
no, didn't get this warning log, our deployed cluster version is 1.16.

I checked the source code,  https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/volume_linux.go#L49 was added at 1.17. Unlucky!  :grinning:

but, I insist that adding a log print after mount success is more helpful when we debug volume issues. 

Thx.

Reply:
Can you also update the log with what volume was just mounted? Does `output` contain anything useful?

Reply:
yes, `output` will show the mount point.
with this log, we can make sure this mount operation has been finished, and we also can calculate how long the mount operation will take.

Reply:
No, output does not show mount point. Here is what gets logged:

> 1213 10:44:14.737252  545222 mount_linux.go:199] Volume successfully mounted, output: ()

Which is expected, because a successful mount command does not return anything.